singleheader/amalgamate.sh

```diff
@@ -108,7 +108,7 @@ mkdir -p $AMALGAMATE_OUTPUT_PATH
 AMAL_H="${AMALGAMATE_OUTPUT_PATH}/simdjson.h"
 AMAL_C="${AMALGAMATE_OUTPUT_PATH}/simdjson.cpp"
 DEMOCPP="${AMALGAMATE_OUTPUT_PATH}/amalgamate_demo.cpp"
-README="$AMALGAMATE_OUTPUT_PATH/README.md"
```

Comment:
There's a line at the bottom, `cat $README`, which will now read `cat`--won't this cause amalgamate to block?

I'd be OK with removing the instructions at the bottom. I don't really think users should be calling this themselves, anyway. However, I think @lemire had some concerns about changing amalgamate too much from an earlier attempt at this, and I can't remember what they were ...

Reply:
I think I was preoccupied with testing. People may complain that the amalgamate_demo.cpp thing is useless, but it has a purpose.

Reply:
Ah! Do you have an objection to keeping these files in the repository but not regenerating them from amalgamate.sh, then?

Reply:
"Ah! Do you have an objection to keeping these files in the repository but not regenerating them from amalgamate.sh" 

I do not.